Documentation/Getting-Started/glossary.md

```diff
@@ -98,12 +98,6 @@ A stretched cluster is a deployment model in which two datacenters with low late
 
 The [Rook toolbox](../Troubleshooting/ceph-toolbox.md) is a container with common tools used for rook debugging and testing.
 
-## Koor 
```

Comment:
is removing this a mistake/merge strategy thing or is it because we want them in Kbase? @galexrt

Reply:
This seems like a mistake/ merge issue. I'll look into fixing that